modules/serverless-python-function-return-values.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-python-functions.adoc
+
+[id="serverless-python-function-return-values_{context}"]
+= Python function return values
+
+Functions may return any value supported by https://flask.palletsprojects.com/en/1.1.x/quickstart/#about-responses[Flask] because the invocation framework proxies these values directly to the Flask server.
+
+.Example
+[source,python]
+----
+def main(context: Context):
+    data = { "message": "Howdy!" }
+    headers = { "content-type": "application/json" }
+    return body, 200, headers
```

Comment:
I'm not a much familiar with python and Flask api, but I think this will not work since `body` was not defined. 
I tried that I got an error. 

So maybe it would be good to change to:

`body = { "message": "Howdy!" }`
or
`return data, 200, headers`

Reply:
@lance wdyt?

Reply:
Oh yeah - good catch. Setting `body` is probably the more illustrative approach. `body = { "message": "Howdy!" }`

Reply:
Adding in the next commit, lmk if it looks correct 🙂